src/preferences/dialog/dlgprefinterface.cpp

```diff
@@ -4,12 +4,14 @@
 #include <QList>
 #include <QLocale>
 #include <QScreen>
+#include <QVariant>
 #include <QtGlobal>
 
 #include "control/controlobject.h"
 #include "control/controlproxy.h"
 #include "defs_urls.h"
 #include "moc_dlgprefinterface.cpp"
+#include "preferences/constants.h"
```

Comment:
can be moved to `preferences/dialog/dlgpreferencepage.h` to include it only once

Reply:
... and be removed from `src/mixxxmainwindow.cpp`

Reply:
are you sure about dlgpreferencepage.h? its not used in there anywhere.

Reply:
That's the baseclass of all pref pages, would it increase the include 'load' including it there?
If so, just leave as it is.

Reply:
not really, but its bad practice as well. It also wouldn't make any difference in terms of compilation time. In general its best to include what you use at the point of usage and nowhere else...

Reply:
I'm okay with removing it from mixxxmainwindow.cpp, though it doesn't hurt to keep either. Doesn't really go against good practices either since it is used in that file...

Reply:
okay. it's already included in the header

Reply:
yeah, so the cppcoreguidelines don't say much about that. If you insist, I'll remove the header from the `.cpp`.

Reply:
Yes, please remove it. It'd be removed anyway with the next include cleanup.

Reply:
Umm, this got a little mixed up, I was referring to the include in `src/mixxxmainwindow.cpp`.
Anyway, clean this up as much you like, I don't mind too much, just noticed it when looking for places where constants.h is included.

Reply:
yup sorry. I don't think anymore cleanup is needed now.